src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManagerBase.cs

```diff
@@ -253,6 +255,18 @@ protected static bool IsInvalidArgument(IReadOnlyList<object> list)
             {
                 await task(message);
             }
+            catch (TimeoutException ex)
+            {
+                // Regard the case connection already disconnected when send group message got timeout as success
+                if (message is IAckableMessageWithConnectionId msg && !_clientConnectionManager.ClientConnections.TryGetValue(msg.ConnectionId, out var connection))
+                {
+                    Log.SucceededToSendMessage(Logger, message);
```

Comment:
don't log success but just ignore?

Reply:
I need it for message log. will check whether we need to actual log it inside.